src/metrics/mongoose_metrics_probe_node_sessions.erl

```diff
@@ -14,12 +14,11 @@
 %% limitations under the License.
 %%==============================================================================
 -module(mongoose_metrics_probe_node_sessions).
--behaviour(mongoose_metrics_probe).
+-behaviour(mongoose_instrument_probe).
```

Comment:
We need to rename and move all three probe modules, because the goal is to empty the whole `mongoose_metrics` directory. We also don't want to give an impression, that this is still the old probe.